sdk/lusid/utilities/api_client_builder.py

```diff
@@ -85,6 +87,14 @@ def build(cls, api_secrets_filename=None, id_provider_response_handler=None, api
         config.access_token = api_token
         config.host = configuration.api_url
 
+        if api_configuration.flag_to_use_decimal is not None:
+            config.flag_to_use_decimal = api_configuration.flag_to_use_decimal
```

Comment:
You should edit the configuration in the lusid-sdk-generators repo to add a default param for flag_to_use_decimal just after the tcp keep alive one - https://github.com/finbourne/lusid-sdk-generators/blob/master/python/generate/templates/configuration.mustache#L310